airflow/templates/configmaps.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ .Values.airflow.prefix -}}airflow-cfg"
+data:
+  AIRFLOW_HOME: "/usr/local/airflow"
+  {{- if .Values.airflow.fernet_key }}
+  FERNET_KEY: "{{- .Values.airflow.fernet_key -}}"
+  {{- end }}
+  RABBITMQ_HOST: "{{- .Values.airflow.prefix -}}{{- .Values.db.rabbitmq.basename -}}"
```

Comment:
Typo: `hostname` instead of `basename`

Reply:
I wanted to emphasis it is not a full hostname. the final hostname will be prefix with airflow.prefix

Reply:
In that case shouldn't the corresponding variable in `values.yaml` be using `basename` instead of `hostname`

Reply:
hum this part works pretty well, I don't see your point. values.yaml allows user to specify the basehostname, and here it prefixed with the generic prefix (I use it so it can be possible to start several airflow instance in the same namespace)